### ISSUE TITLE
CMake build issue on Ubuntu 16.06, arm64

### ISSUE THREAD

**Issue by rytrose** [YannickJadoul/Parselmouth#9]
I'm looking to use Parselmouth on an NVIDIA Jetson TX2, but I'm running into this CMake error. I updated CMake to 13.3.1. The traceback from `sudo pip install praat-parselmouth` is as follows:

```
Installing collected packages: praat-parselmouth
  Running setup.py install for praat-parselmouth ... error
    Complete output from command /usr/bin/python3.6 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-zb2h1v06/praat-parselmouth/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-dt2lxbst/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_ext
    -- The C compiler identification is GNU 6.5.0
    -- The CXX compiler identification is GNU 6.5.0
    -- Check for working C compiler: /usr/bin/cc
    -- Check for working C compiler: /usr/bin/cc -- works
    -- Detecting C compiler ABI info
    -- Detecting C compiler ABI info - done
    -- Detecting C compile features
    -- Detecting C compile features - done
    -- Check for working CXX compiler: /usr/bin/c++
    -- Check for working CXX compiler: /usr/bin/c++ -- works
    -- Detecting CXX compiler ABI info
    -- Detecting CXX compiler ABI info - done
    -- Detecting CXX compile features
    -- Detecting CXX compile features - done
    -- Found PythonInterp: /usr/bin/python3.6 (found version "3.6.7")
    -- Found PythonLibs: /usr/lib/aarch64-linux-gnu/libpython3.6m.so
    -- Performing Test HAS_CPP14_FLAG
    -- Performing Test HAS_CPP14_FLAG - Success
    -- pybind11 v2.2.3
    -- Performing Test HAS_FLTO
    -- Performing Test HAS_FLTO - Success
    -- LTO enabled
    CMake Error at CMakeLists.txt:46 (add_subdirectory):
      add_subdirectory given source "tests" which is not an existing directory.
```

**Comment by YannickJadoul** [comment]
@rytrose Argh. Yes, my bad. There is a problem in the latest release's (0.3.2) source distribution of Parselmouth. I forgot to include the test directory in [`MANIFEST.in`](https://github.com/YannickJadoul/Parselmouth/blob/03e2546a4f42aaeb5b6e69566c1f996deb4e89dc/MANIFEST.in), which is why it is not in the `tar.gz` pip downloads but referenced in the `CMakeLists.txt`

It's been fixed [on the master branch](https://github.com/YannickJadoul/Parselmouth/blob/master/MANIFEST.in) since I discovered this issue, but since I hope to release 0.4.0 soon, I had hoped I could get away with it.

So, a quick solution is downloading the source of [0.3.2](https://github.com/YannickJadoul/Parselmouth/releases/tag/v0.3.2) ([tar.gz](https://github.com/YannickJadoul/Parselmouth/archive/v0.3.2.tar.gz) or [zip](https://github.com/YannickJadoul/Parselmouth/archive/v0.3.2.zip)) and running `pip install .` in the root directory. (Or of course the master branch, if you feel adventurous and want to use some new features.)

Apart from that: if you want/insist, I could draw up a version 0.3.3 that has this issue fixed.

**Comment by rytrose** [comment]
Hey @YannickJadoul, thanks for the super speedy reply :) I can definitely leverage your solutions, and leave release versioning up to you. I'll let you know if I have any other issues that would want be to insist lol.

**Comment by YannickJadoul** [comment]
@rytrose I just found out that I can make a 'post-release', that can fix this kind of packaging errors. So I'm now seeing if I can do this, such that I don't have to release 0.3.3, but rather 0.3.2.post1.

**Comment by rytrose** [comment]
Awesome! `pip install .` on `master` worked for me too, thanks!

**Comment by YannickJadoul** [comment]
Cool, that post-release seems to be working, now :-) `pip install praat-parselmouth` will now also result in an installable source dist.

I'll close this issue; thanks for reporting!

PS: The compilation of Praat takes quite some time and there are unfortunately no ARM binary wheels available. If you need to install multiple times on the same architecture you might find the `pip wheel praat-parselmouth` command quite useful, creating your own pre-compiled wheel that you can install.

**Comment by YannickJadoul** [comment]
Update: Sorry, it seems that the post-release is still shown as a separate release, so rather than causing confusion, I removed it again. Will be fixed soon enough by 0.4.0 anyway.
